src/evm/mod.rs

```diff
@@ -1865,6 +1865,37 @@ pub fn evm_direct_deploy_and_compressed_call_add(log_to: Option<&Path>, debug: b
     }
 }
 
+#[test]
+fn evm_reverter_factory_test() {
+    _evm_reverter_factory_test_impl();
+}
+
+fn _evm_reverter_factory_test_impl() {
+    let rt_env = RuntimeEnvironment::new(Uint256::from_usize(1111), None);
+    let mut machine = load_from_file(Path::new("arb_os/arbos.mexe"), rt_env);
+    machine.start_at_zero();
+
+    let _contract = match AbiForContract::new_from_file(&test_contract_path("ReverterFactory")) {
```

Comment:
I don't see a `ReverterFactory` contract in the test package. Should that exist?

Reply:
Added it.